### ISSUE TITLE
No need to update annotationManager.originLocation ?

### ISSUE THREAD

**Issue by omarojo** [mapbox/mapbox-arkit-ios#33]
Once we ask the user for the current location, and we set that location to `annotationManager.originLocation` there is no need to keep updating the current location ?

**Comment by francisjervis** [comment]
No, and doing so will increase power consumption. However, you need to balance this with accurately positioning content, relative to the device. If you think your users will be moving a considerable distance (eg for a navigation or city information app), you should consider updating the location while the AR view is active. You should ensure that your app stops updating the location when the AR view controller is not in use, however.

**Comment by Michele404** [comment]
Why you should stop update location when AR is running?
